apps/src/gamelab/animationListModule.js

```diff
@@ -336,6 +407,46 @@ export function addLibraryAnimation(props) {
 }
 
 /**
+ * User selects new frames to add to the animation. Set these as pending
+ * before loading them into Piskel.
+ * @param {!AnimationKey} key
+ * @param {AnimationProps} props
+ * @returns {{type: string, key: AnimationKey, props: AnimationProps}}
+ */
+export function setPendingAnimationAddition(key, props) {
+  return {
+    type: SET_PENDING_ANIMATION_ADDITION,
+    key,
+    props
+  };
+}
+
+/**
+ * After an animation addition is added, remove it as pending.
+ * @returns {{type: string}}
```

Comment:
Not strictly true. We're returning a function, that eventually dispatches an object with type: string